### PR TITLE
[RFR] Fix Day trader analysis failure

### PR DESCRIPTION
**Pull request by nachandr**

### CODE REVIEW

**Comment by nachandr** [comment]
Daytrader_api_oct11

Screen shot of Jenkins run that shows that the analysis of the day trader app passed

**Review by mguetta1 — COMMENTED** [review]
Hi @nachandr thanks for the PR. Could you please add issue incidents as well?

**Comment by nachandr** [comment]
> Hi @nachandr thanks for the PR. Could you please add issue incidents as well?

Issue incidents have been added.

File: analysis/tc_daytrader.go [hunk 1]
```diff
@@ -20,31 +20,96 @@ var Daytrader = TC{
...
+				Category: "mandatory",
+				Effort: 1,
+				Description: "File system - Java IO",
+				RuleSet: "cloud-readiness",
+				Rule: "local-storage-00001",
+				Incidents: []api.Incident{
+					{
+						File: "/shared/source/sample/daytrader-ee7-web/src/main/java/com/ibm/websphere/samples/daytrader/web/TradeScenarioServlet.java",
+						Line: 125,
+					},
```
Comment (mguetta1) [inline analysis/tc_daytrader.go:39]: We usually check also incident message, could you please add it?

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by aufi — APPROVED** [review]
LGTM

**Review by mguetta1 — APPROVED** [review]

**Pull request merged**